apps/condo/domains/meter/gql.js

```diff
@@ -10,13 +10,13 @@ const { generateGqlQueries } = require('@condo/domains/common/utils/codegenerati
 
 const COMMON_FIELDS = 'id dv sender { dv fingerprint } v deletedAt newId createdBy { id name } updatedBy { id name } createdAt updatedAt'
 
-const METER_RESOURCE_FIELDS = `{ name measure ${COMMON_FIELDS} }`
+const METER_RESOURCE_FIELDS = `{ name nameNonLocalized measure measureNonLocalized ${COMMON_FIELDS} }`
 const MeterResource = generateGqlQueries('MeterResource', METER_RESOURCE_FIELDS)
 
-const METER_READING_SOURCE_FIELDS = `{ type name ${COMMON_FIELDS} }`
+const METER_READING_SOURCE_FIELDS = `{ type name nameNonLocalized ${COMMON_FIELDS} }`
 const MeterReadingSource = generateGqlQueries('MeterReadingSource', METER_READING_SOURCE_FIELDS)
 
-const METER_FIELDS = `{ number numberOfTariffs installationDate commissioningDate verificationDate nextVerificationDate controlReadingsDate sealingDate accountNumber organization { id } property { id address addressMeta { ${ADDRESS_META_SUBFIELDS_QUERY_LIST} } } unitName unitType place resource { id name measure } ${COMMON_FIELDS} }`
+const METER_FIELDS = `{ number numberOfTariffs installationDate commissioningDate verificationDate nextVerificationDate controlReadingsDate sealingDate accountNumber organization { id } property { id address addressMeta { ${ADDRESS_META_SUBFIELDS_QUERY_LIST} } } unitName unitType place resource { id name nameNonLocalized measure measureNonLocalized } ${COMMON_FIELDS} }`
```

Comment:
It’s better to reuse meter reading resource and source field

Reply:
done